packages/language-server/src/xml-view-diagnostics.ts

```diff
@@ -67,7 +71,7 @@ function validationIssuesToLspDiagnostics(
       case "NonStableIDIssue":
         return {
           ...commonDiagnosticPros,
-          code: 1000,
+          code: validations.NON_STABLE_ID.code,
```

Comment:
Line 91: Select a unique ID. The current ID has already been used,
[Is this correct?]